build/bin.NOTICE

```diff
@@ -77,9 +84,6 @@ Apache XML Commons Resolver (lib/xml-resolver.xml-resolver-<VERSION>.jar)
   Apache XML Commons Resolver
   Copyright 2006 The Apache Software Foundation.
 
-  This product includes software developed at
-  The Apache Software Foundation http://www.apache.org/
-

```

Comment:
This line still exists in a few places. This can be removed in all places except for the very first one at the top.

Reply:
I've removed these lines in my edits.